proof/invariant-abstract/DetSchedInvs_AI.thy

```diff
@@ -916,6 +917,54 @@ global_interpretation ep_recv_qs:
   opt_map_opt_map_cons_def_locale _ ep_of eps_of_kh Endpoint recv_q_of RecvEP ep_recv_qs_of_eps
   using ep_recv_qs_of_eps_def recv_q_of_None recv_q_of_Some by unfold_locales
 
+\<comment> \<open>Notifications\<close>
+
+\<comment> \<open>Project notifications from the kernel heap\<close>
+
+primrec ntfn_of :: "Structures_A.kernel_object \<rightharpoonup> notification" where
```

Comment:
It's a good idea to add these, but also as definitions, and with equality setup for the automation lemmas (if that works, I haven't tried out if equality is actually possible)